pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/DineRettigheterOgPlikterAlder.kt

```diff
@@ -0,0 +1,185 @@
+package no.nav.pensjon.brev.maler.vedlegg
```

Comment:
flyttet fra OrienteringOmRettigheter (orientering om rettigheter er gammelt navn for vedlegget)